Patterns/Chinese/Chinese-DateTime.yaml

```diff
@@ -94,7 +94,8 @@ BeforeRegex: !simpleRegex
   def: 以前|之前|前
 AfterRegex: !simpleRegex
   def: 以后|以後|之后|之後|后|後
-# (农历)?(2016年)?一月三日(星期三)?
```

Comment:
Isn't this comment for DateRegexList1? Why delete it?

Reply:
mistake, added back.